### PR TITLE
Refactored

### PR DESCRIPTION
**Pull request by BELOVEZHIE**
https://trello.com/c/wKmzv7CC/250-entity-chevron-inputcopy

### CODE REVIEW

File: src/test/java/EntityChevronTest.java [hunk 1]
```diff
@@ -29,35 +30,29 @@ public void findChevron() throws InterruptedException {
...
-        WebDriverWait wait = new WebDriverWait(driver, 6);
-
-        WebElement findFulfillmentAgain = driver.findElement(By.xpath("//td//div[contains(text(), 'Fulfillment')]"));
+        WebElement findFulfillmentAgain = driver.findElement(expectedSign);
         ProjectUtils.click(driver, findFulfillmentAgain);
 
         WebElement recheckFulfillment = driver.findElement(By.xpath("//a[@class = 'pa-chev-active']"));
-        String ExpectedSign = "Fulfillment";
-        Assert.assertEquals(ExpectedSign, recheckFulfillment.getText());
+        Assert.assertEquals(CHECKING_SIGN, recheckFulfillment.getText());
```
Comment (Stan575) [inline src/test/java/EntityChevronTest.java:55]: switch actual and expected parameters : 
    public static void assertEquals(String actual, String expected)

File: src/test/java/EntityChevronTest.java [hunk 2]
```diff
@@ -73,4 +68,3 @@ public void findChevron() throws InterruptedException {
```
Comment (Stan575) [inline src/test/java/EntityChevronTest.java:70]: delete empty lines

File: src/test/java/EntityChevronTest.java [hunk 3]
```diff
@@ -29,35 +30,29 @@ public void findChevron() throws InterruptedException {
...
 
+        getWebDriverWait().until(driver1 -> fillTime.isDisplayed());
+
         WebElement buttonSaveClick = driver.findElement(By.xpath("//button[@class = 'btn btn-warning']"));
         ProjectUtils.click(driver, buttonSaveClick);
 
-        Assert.assertEquals(driver.findElement(By.xpath("//div[contains(text(),'Fulfillment')]")).getText(),
-                "Fulfillment");
+        Assert.assertEquals(driver.findElement(expectedSign).getText(),
+                CHECKING_SIGN);
```
Comment (Stan575) [inline src/test/java/EntityChevronTest.java:49]: can be one line

File: src/test/java/EntityChevronTest.java [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
 import org.openqa.selenium.*;
-import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.annotations.Test;
 import org.testng.Assert;
 import runner.ProjectUtils;
 
 
 public class EntityChevronTest extends BaseTest {
 
+    private final String CHECKING_SIGN = "Fulfillment";
```
Comment (Stan575) [inline src/test/java/EntityChevronTest.java:10]: should be : private static final

**Review by Stan575 — CHANGES_REQUESTED** [review]

**Comment by guram21** [comment]
This PR is no longer relevant

**Pull request closed without merging**